### PR TITLE
Second task commit. Waiting for review.

### PR DESCRIPTION
**Pull request by Radu-Stroe**
Verify Terms & Conditions can be accessed via subscriber page
    Given I visit the Subscriptions page
    When I click T&C
    Then I expect T&C content is displayed

### CODE REVIEW

File: src/test/java/internship/tests/TermsAndConditionsTest.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+public class TermsAndConditionsTest extends TestBase {
+
+    @Test(testName = "Terms and Conditions Test",
+    description = "Testing Terms and Conditions")
+    public void checkTermsAndConditions() {
+        CustomerSubscribePage customerSubscribePage = new CustomerSubscribePage().openPage();
+        customerSubscribePage.clickTermsAndConditionsButton();
+        TermsAndConditionsPage termsAndConditionsPage = new TermsAndConditionsPage();
+        softAssert.get().assertTrue(termsAndConditionsPage.isTermsAndConditionsDisplayed().contains("Terms and Conditions"),
```
Comment (cristinanedin) [inline src/test/java/internship/tests/TermsAndConditionsTest.java]: please use assertEquals

File: src/test/java/internship/framework/pages/TermsAndConditionsPage.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+public class TermsAndConditionsPage extends PageBase {
+
+    @FindBy(xpath = "//h1[@class='headline__heading']")
+    private WebElement termsAndConditions;
+
+    @FindBy(xpath = "//b[contains(text(),'Policies and Processes')]")
+    private WebElement policiesAndProcesses;
+
+    public String isTermsAndConditionsDisplayed()
```
Comment (cristinanedin) [inline src/test/java/internship/framework/pages/TermsAndConditionsPage.java]: a better naming would be getTermsAndConditionsTitleText

File: src/test/java/internship/framework/pages/TermsAndConditionsPage.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    {
+        ProjectLogger.info("Check if Terms and Conditions element is displayed.");
+        return  getElementText(termsAndConditions);
+    }
+
+    public String isPoliciesAndProcessesDisplayed()
```
Comment (cristinanedin) [inline src/test/java/internship/framework/pages/TermsAndConditionsPage.java]: same as above..

**Review by cristinanedin — CHANGES_REQUESTED** [review]

File: src/test/java/internship/tests/TermsAndConditionsTest.java (hunk 1 above)
Comment (Radu-Stroe) [reply src/test/java/internship/tests/TermsAndConditionsTest.java]: Fixed.

File: src/test/java/internship/framework/pages/TermsAndConditionsPage.java (hunk 2 above)
Comment (Radu-Stroe) [reply src/test/java/internship/framework/pages/TermsAndConditionsPage.java]: Fixed.

File: src/test/java/internship/framework/pages/TermsAndConditionsPage.java (hunk 3 above)
Comment (Radu-Stroe) [reply src/test/java/internship/framework/pages/TermsAndConditionsPage.java]: Fixed.

File: src/test/java/internship/framework/pages/TermsAndConditionsPage.java [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+    @FindBy(xpath = "//h1[@class='headline__heading']")
+    private WebElement termsAndConditionsTitleText;
+
+    @FindBy(xpath = "//b[contains(text(),'Policies and Processes')]")
+    private WebElement policiesAndProcessesChapterTitleText;
+
+    public String getTermsAndConditionsTitleText()
+    {
+        ProjectLogger.info("Check if Terms and Conditions element is displayed.");
```
Comment (cristinanedin) [inline src/test/java/internship/framework/pages/TermsAndConditionsPage.java]: please log the proper scope of the method

Comment (Radu-Stroe) [reply src/test/java/internship/framework/pages/TermsAndConditionsPage.java]: Fixed.

File: src/test/java/internship/tests/TermsAndConditionsTest.java [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+public class TermsAndConditionsTest extends TestBase {
+
+    @Test(testName = "Terms and Conditions Test",
+    description = "Testing Terms and Conditions")
+    public void checkTermsAndConditions() {
+        CustomerSubscribePage customerSubscribePage = new CustomerSubscribePage().openPage();
+        customerSubscribePage.clickTermsAndConditionsButton();
+        TermsAndConditionsPage termsAndConditionsPage = new TermsAndConditionsPage();
+        softAssert.get().assertEquals(termsAndConditionsPage.getTermsAndConditionsTitleText(),"Terms and Conditions",
```
Comment (cristinanedin) [inline src/test/java/internship/tests/TermsAndConditionsTest.java]: please extract "Terms and Conditions" as a constant

File: src/test/java/internship/tests/TermsAndConditionsTest.java [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+                "Terms and Conditions element is not displayed.");
+        softAssert.get().assertEquals(termsAndConditionsPage.getPoliciesAndProcessesChapterText(), "Policies and Processes",
```
Comment (cristinanedin) [inline src/test/java/internship/tests/TermsAndConditionsTest.java]: same as above

**Review by cristinanedin — CHANGES_REQUESTED** [review]

File: src/test/java/internship/tests/TermsAndConditionsTest.java (hunk 5 above)
Comment (Radu-Stroe) [reply src/test/java/internship/tests/TermsAndConditionsTest.java]: Fixed.

File: src/test/java/internship/tests/TermsAndConditionsTest.java (hunk 6 above)
Comment (Radu-Stroe) [reply src/test/java/internship/tests/TermsAndConditionsTest.java]: Fixed.

**Review by cristinanedin — APPROVED** [review]

**Pull request merged**